app/models/manageiq/providers/openstack/infra_discovery.rb

```diff
@@ -3,15 +3,15 @@
 module ManageIQ::Providers::Openstack
   class InfraDiscovery
     def self.probe(ost)
-      res = ""
       if ManageIQ::NetworkDiscovery::PortScanner.portOpen(ost, 6385)
+        res = ""
         Socket.tcp(ost.ipaddr, 6385) do |s|
```

Comment:
Hi @gildub, we found the issue, when SSL is enabled, port used by ironic from public network is 13385 - we should check this port too.


| ironic           | baremetal               | regionOne                                                                     |
|                  |                         |   admin: http://192.168.24.3:6385                                             |
|                  |                         | regionOne                                                                     |
|                  |                         |   internal: http://192.168.24.3:6385                                          |
|                  |                         | regionOne                                                                     |
|                  |                         |   public: https://192.168.24.2:13385

Reply:
Thanks Gilles!